aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -731,35 +731,35 @@ module std::features {
     /// Check whether the feature is enabled.
     public fun is_enabled(feature: u64): bool acquires Features {
         exists<Features>(@std) &&
-            contains(&borrow_global<Features>(@std).features, feature)
+            contains(&Features[@std].features, feature)
     }
 
     /// Helper to include or exclude a feature flag.
     fun set(features: &mut vector<u8>, feature: u64, include: bool) {
         let byte_index = feature / 8;
         let bit_mask = 1 << ((feature % 8) as u8);
-        while (vector::length(features) <= byte_index) {
-            vector::push_back(features, 0)
+        while (features.length() <= byte_index) {
+            features.push_back(0)
         };
-        let entry = vector::borrow_mut(features, byte_index);
+        let entry = features.borrow_mut(byte_index);
```

Comment:
Could use `&mut features[byte_index]` instead.

Optionally, the below uses of `*entry` could be replaced with `features[byte_index]` instead.